airbyte-integrations/connectors/source-amplitude/source_amplitude/manifest.yaml

```diff
@@ -0,0 +1,171 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_pointer:
+        - "{{ options.get('data_field') }}"
+  requester:
+    type: HttpRequester
+    url_base: "https://{{'analytics.eu.' if config['data_region'] == 'EU Residency Server' else '' }}amplitude.com/api"
+    http_method: "GET"
+    authenticator:
+      type: BasicHttpAuthenticator
+      username: "{{ config['api_key'] }}"
+      password: "{{ config['secret_key'] }}"
+    error_handler:
+      type: DefaultErrorHandler
+      response_filters:
+        - http_codes: [ 400 ]
+          action: IGNORE
+          error_message: The file size of the exported data is too large. Shorten the time ranges and try again. The limit size is 4GB.
+        - http_codes: [ 404 ]
+          action: IGNORE
+          error_message: No data collected
+        - http_codes: [ 504 ]
+          action: IGNORE
+          error_message: The amount of data is large causing a timeout. For large amounts of data, the Amazon S3 destination is recommended.
+
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "*ref(definitions.requester)"
+
+  date_stream_slicer:
+    type: DatetimeStreamSlicer
+    start_datetime:
+      datetime: "{{ format_datetime(config['start_date'], '%Y-%m-%d') }}"
+    end_datetime:
+      datetime: "{{ now_utc().strftime('%Y-%m-%d') }}"
+    datetime_format: "%Y-%m-%d"
+    cursor_granularity: P1D
+    step: P15D
+    cursor_field: "{{ options['stream_cursor_field'] }}"
+
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+
+  annotations_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "annotations"
+      primary_key: "id"
+      path: "/2/annotations"
+      data_field: "data"
+
+  cohorts_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "cohorts"
+      primary_key: "id"
+      path: "/3/cohorts"
+      data_field: "cohorts"
+
+  base_incremental_stream:
+    checkpoint_interval: 10
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      requester:
+        $ref: "*ref(definitions.requester)"
+        request_options_provider:
+          request_parameters:
+            start: "{{format_datetime(stream_slice.start_time, '%Y%m%d') }}"
+            end: "{{format_datetime(stream_slice.end_time, '%Y%m%d') }}"
+      stream_slicer:
+        $ref: "*ref(definitions.date_stream_slicer)"
```

Comment:
Using the latest version of the airbyte-cdk, this should be under `incremental_sync` which is directly under the stream

Reply:
updated